indra/tests/test_pybel_assembler.py

```diff
@@ -0,0 +1,33 @@
+from indra.assemblers import pybel_assembler as pa
+from indra.statements import *
+import pybel
+
+def test_simple_phosphorylation_no_evidence():
+    braf = Agent('BRAF', db_refs={'HGNC': '1097', 'UP': 'P15056'})
+    mek = Agent('MAP2K1', db_refs={'HGNC': '6840', 'UP': 'Q02750'})
+    stmt = Phosphorylation(braf, mek, 'S', '218')
+    pba = pa.PybelAssembler([stmt])
+    belgraph = pba.make_model()
+    pybel.to_bel(belgraph)
+    assert len(belgraph.nodes()) == 2
+    assert ('Protein', 'HGNC', 'BRAF') in belgraph.nodes()
+    assert ('Protein', 'HGNC', 'MAP2K1') in belgraph.nodes()
+    assert len(belgraph.edges()) == 1
+
+def test_simple_phosphorylation_with_evidence():
+    braf = Agent('BRAF', db_refs={'HGNC': '1097', 'UP': 'P15056'})
+    mek = Agent('MAP2K1', db_refs={'HGNC': '6840', 'UP': 'Q02750'})
+    ev1 = Evidence('reach', '1234', '12345678', 'Some example text.')
+    ev2 = Evidence('reach', '2345', '23456789', 'Some more example text.')
+    stmt = Phosphorylation(braf, mek, 'S', '218', evidence=[ev1, ev2])
+    pba = pa.PybelAssembler([stmt])
+    belgraph = pba.make_model()
+    pybel.to_bel(belgraph)
+    assert len(belgraph.nodes()) == 2
```

Comment:
PyBEL stores a different node for each variant of the original agent - there should also be another node for `(pc.PROTEIN, 'HGNC', 'MAPK1', (pc.PMOD, (pc.BEL_DEFAULT_NAMESPACE, 'Ph')))`.

We also have to consider if we want to store this information in BEL as `act(p(HGNC:BRAF)) => p(HGNC:MAP2K1, pmod(Ph, 'Ser', 218)` or `act(p(HGNC:BRAF)) => rxn(reactants(p(HGNC:MAP2K1)), products(p(HGNC:MAP2K1, pmod(Ph, 'Ser', 218))`. I personally prefer the latter but for different problems it might better to represent it with the former.

In the latter case, we also have to add a node for `(pc.REACTION, ((pc.PROTEIN, 'HGNC', 'MAPK1'),), ((pc.PROTEIN, 'HGNC', 'MAPK1',  (pc.PMOD, (pc.BEL_DEFAULT_NAMESPACE, 'Ph'), )), ))` however it looks like we're getting to an edge case in re-usability in the PyBEL data structure, since the node tuples are a bit redundant of the associated data dictionaries. We'll have to investigate further